src/kitovu/cli.py

```diff
@@ -62,3 +66,51 @@ def validate(config: typing.Optional[pathlib.Path] = None) -> None:
         syncing.validate_config(config, CliReporter())
     except utils.UsageError as ex:
         raise click.ClickException(str(ex))
+
+
+@cli.command()
+def docs() -> None:
+    """Open the documentation in the browser."""
+    # FIXME: make version aware
+    webbrowser.open_new_tab('https://kitovu.readthedocs.io/en/latest')
+
+
+DEFAULT_EDITORS = [
+    'vim',
+    'emacs',
+    'nano',
+    'editor',
+    'notepad',
+]
+
+
+@cli.command()
+@click.option('--config', type=pathlib.Path, help="The configuration file to edit")
+@click.option('--editor', type=str, help=f"The command of the editor to use. \
```

Comment:
I'd prefer ending the string there (with `"`) instead of the backslash, and then opening a new string (via `f"`) on the next line. Python will automatically concatenate them, and it's less brittle than the backslash (when e.g. adding a space or whatever).